packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -120,8 +120,8 @@ done
 #########################
 # kontrol prove options #
 #########################
-max_depth=1000000
-max_iterations=1000000
+max_depth=10000
+max_iterations=10000
```

Comment:
Why were these changed and how do you know what to set them at in the first place? I know you said you were working on some documentation for all these options, so perhaps that will answer this

Reply:
These were changed because the previous value was set to not produce any nodes while executing the summary contract with Kontrol. 10000 is sort of the standard that we use. When observing the KCFGs, usually the steps between the nodes never exceed 10k (~8k is the max I've observed for our proofs). This means that if we were to set a lower value (e.g., 1k) Kontrol would create nodes after that many steps of execution, but this would increase execution time. Here, the rationale is that there are not more than 10k steps between one node and the following in normal conditions. We set it to 10k as a way of measuring and inspecting when this happens. [Here's the documentation](https://docs.runtimeverification.com/kontrol/cheatsheets/kontrol-cheatsheet#flags) we have on these flags so far, which I feel might be improved. Any feedback welcome!

Reply:
I see, thanks for explaining! I read the linked docs—can we add `--counterexample-information` as a flag? I recall that UX was very helpful for failures, but unsure if there's any downsides to using that as the default

Reply:
Oh perfect! Though I don't see a `--no-counterexample-information` in the linked docs 

Either way will resolve this thread